src/test/java/core/basesyntax/dao/FruitDaoImplTest.java

```diff
@@ -0,0 +1,46 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.db.FruitStorage;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FruitDaoImplTest {
+    private static FruitDao fruitDao;
+    private static Map<String,Integer> expectedStorage;
+
+    @BeforeClass
+    public static void beforeClass() {
+        fruitDao = new FruitDaoImpl();
+        expectedStorage = new HashMap<>();
+        expectedStorage.put("banana",555);
+        expectedStorage.put("apple",111);
+    }
+
+    @Test
+    public void putAndGetIs_Ok() {
+        fruitDao.put("apple",111);
```

Comment:
mentors were saying that we cant use our implemented solutions to access our storage\db, so we must do it like this:
FruitStorage.storage.put()
without using our dao implementation (check it everywhere)